dist/sharect.babel.js

```diff
@@ -27,6 +27,20 @@ var Sharect = function () {
     var _top = 0;
     var _left = 0;
 
+    function offsetParent(el) {
```

Comment:
This is basically https://api.jquery.com/offsetParent/ from https://github.com/jquery/jquery/blob/56136897f241db22560b58c3518578ca1453d5c7/src/offset.js#L162-L172